platforms-serverless/firebase-functions/functions/package.json

```diff
@@ -8,7 +8,7 @@
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
     "logs": "firebase functions:log",
-    "postinstall": "CI=1 && PRISMA_TELEMETRY_INFORMATION='ecosystem-tests platforms/firebase-functions postinstall' && yarn prisma generate"
+    "gcp-build": "CI=1 && PRISMA_TELEMETRY_INFORMATION='ecosystem-tests platforms/firebase-functions postinstall' && npx prisma generate"
```

Comment:
Uh, platform specific build command?
Should probably document if it is _needed_ for anything.

Also:
```suggestion
    "gcp-build": "CI=1 && PRISMA_TELEMETRY_INFORMATION='ecosystem-tests platforms/firebase-functions gcp-build' && npx prisma generate"
```

Reply:
Yep, it has changed in the new versions, same for firebase.

Reply:
new versions of firebase/gcp?

Reply:
Yes. Very hard to find in their docs.

Reply:
(can this be marked as resolved?)